README.md

```diff
@@ -36,3 +36,199 @@ Flags:
 
 Use "vault-unsealer [command] --help" for more information about a command.
 ```
+## Setup AWS
+1. Create or use a KMS key in the region you want:
+https://console.aws.amazon.com/iam/home?region=us-east-1#/encryptionKeys/us-east-1
+
+2.note the alias name of the key for example:
+`alias/vault`
+
+3.note the ARN for the key.
+
+3. Add the following IAM permissions to the identity where the vault-unsealer will run 
+for example if running on kubernetes via KOPS run `kops edit cluster` and add the permissions under additional node policies
+
+```yaml
+spec:
+  additionalPolicies:
+  node: |
+  {
+            "Sid": "",
+            "Effect": "Allow",
+            "Action": [
+                "ssm:GetParameters"
+            ],
+            "Resource": [
+                "arn:aws:ssm:<region>:<aws-account-id>:parameter/*"
+            ]
+        },
+        {
+            "Sid": "",
+            "Effect": "Allow",
+            "Action": [
+                "ssm:PutParameter",
+                "ssm:DeleteParameter"
+            ],
+            "Resource": [
+                "arn:aws:ssm:<region>:<aws-account-id>:parameter/kubernetes-*"
+            ]
+        },
+        {
+            "Sid": "",
+            "Effect": "Allow",
+            "Action": [
+                "kms:Get*",
+                "kms:ListKeys",
+                "kms:ListAliases"
+            ],
+            "Resource": [
+                "*"
+            ]
+        },
+        {
+            "Sid": "",
+            "Effect": "Allow",
+            "Action": [
+                "kms:DescribeKey",
+                "kms:Encrypt",
+                "kms:Decrypt"
+            ],
+            "Resource": [
+                "arn:aws:kms:<region>:<aws-account-id>:key/<key-id>"
+            ]
+        },
+```
```

Comment:
These IAM rules allow every pod access to your vault unseal key. I strongly suggest to use kube2iam to only aloow that for a Role that is associated to the vault-unsealer pods

Reply:
sure thanks